<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Titillium Web</family>
    <pointsize>7</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>IIT Delhi VPN</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #ffffff;
    font-family: &quot;Titillium Web&quot;;
    font-size: 7pt;
}
QLabel {
    font-family: &quot;Titillium Web&quot;;
    font-size: 18px;
    font-weight: 500;
    color: #2b2b2b;
    padding-bottom: 4px;
}
QLineEdit {
    font-family: &quot;Titillium Web&quot;;
    font-size: 14px;
    font-weight: 400;

    color: #1a1a1a;
    background-color: #ffffff;

    border: 1px solid #d6d6d6;
    border-radius: 6px;

    padding: 8px 10px;

    selection-background-color: #c21717;
    selection-color: #ffffff;
}

QLineEdit:hover {
    border: 1px solid #c21717;
}

/* Focus state */
QLineEdit:focus {
    border: 1.5px solid #c21717;
    background-color: #ffffff;
}

/* Disabled state */
QLineEdit:disabled {
    background-color: #f4f4f4;
    color: #9a9a9a;
    border: 1px solid #e0e0e0;
}
QLineEdit[error=&quot;true&quot;] {
    border: 1.5px solid #c21717;
    background-color: #fdeaea;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>721</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resources.qrc">:/img/logo.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Titillium Web</family>
           <pointsize>36</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(194, 23, 23);
font: 36pt ;</string>
         </property>
         <property name="text">
          <string>IIT Delhi VPN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Username</string>
             </property>
             <property name="buddy">
              <cstring>usernameEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="usernameEdit">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="cursor">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Password</string>
             </property>
             <property name="buddy">
              <cstring>passwordEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="passwordEdit">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::EchoMode::Password</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Certificate</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="ovpnPathEdit">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="browseOvpn">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgba(0, 0, 0, 10);
padding: 8px 20px;
border: 1px solid  rgba(0, 0, 0, 80);
border-radius: 5px;
font: 11pt;
}
QPushButton:hover{
background-color: rgba(0, 0, 0, 20);
}</string>
               </property>
               <property name="text">
                <string>Browse</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="connectButton">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    font: 12pt &quot;Titillium Web&quot;;
    background-color: rgba(194, 23, 23, 10);
    color: rgb(194, 23, 23);
    border: 1px solid #c21717;
    border-radius: 10px;
    padding: 10px 20px;
}

QPushButton:hover {
	color: rgb(255,255,255);
    background-color: rgba(194, 23, 23, 200);
}
</string>
           </property>
           <property name="text">
            <string>Connect</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>987</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>10</height>
    </size>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
